pmoves/tools/beats_to_voice.py

```
@@ -117,6 +120,70 @@ def _check_flute_health(base_url: str) -> bool:
+            aid = data.get("user_id") or agent_id
+            if not text:
+                return
+            sys.stderr.write(
+                f"[beats_to_voice] Trigger received agent={aid} len={len(text)}\n"
+            )
+            results = run_pipeline(text=text, voice=voice, agent_id=aid, flute_url=flute_url)
+            cgp = results.get("cgp_packet", {})
+            if cgp:
+                await nc.publish(NATS_SUBJECT, json.dumps(cgp).encode("utf-8"))
+                sys.stderr.write(f"[beats_to_voice] CGP published to {NATS_SUBJECT}\n")
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)